.github/CODEOWNERS

```diff
@@ -133,6 +133,7 @@
 /core/templates/dev/head/components/state-editor/ @aks681
 /core/templates/dev/head/services/compute-graph.service.ts @aks681
 /core/templates/dev/head/services/context.service*.ts @aks681
+/core/templates/dev/head/services/tracker.service.ts @aks681
```

Comment:
Let's call this debug-info-tracker -- the current name is too generic. (And rename the rest to DebugInfoTrackerService, etc.)

Reply:
Changed it to debug-info-tracker.service